students/javarb/sessions/2019/april/Session_30.md

```diff
@@ -0,0 +1,108 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 30 (16/04/2019)
+
+- Reviewing sprint project (issue) tasks and solving problems and feedback
+- Reviewing main project tasks and planning next Sprint
```

Comment:
```suggestion
- Reviewing main project tasks and planning next sprint
```
You have written `sprint` with a lower case everywhere else - so this helps with consistency.